plugins/woocommerce/client/blocks/assets/js/base/components/button/index.tsx

```diff
@@ -29,6 +29,10 @@ export interface ButtonProps extends WCButtonProps {
 	 * @default false
 	 */
 	removeTextWrap?: boolean;
+	/**
+	 * Legacy prop that gets filtered out - not used by this component
+	 */
+	showSpinner?: boolean;
```

Comment:
The Points and Rewards triggers a warning for this, so I also fixed it in core

Reply:
https://github.com/woocommerce/woocommerce-points-and-rewards/pull/738